auth-web/src/components/auth/common/PaymentMethods.vue

```diff
@@ -82,6 +82,18 @@
                   />
                 </div>
 
+                <!-- EFT -->
+                <div
```

Comment:
This needs to be feature flagged as well

Reply:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/bc427f7b-bea7-46a7-8a2c-1cd25cdbaa26)